components/tabstrip/tabs-alignment.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Tabs Alignment
+page_title: TabStrip Tabs Alignment
+description: The TabAlignment parameter in the TabStrip component allows you to control the alignment of the tabs. By default, tabs align to the left of the TabStrip header.
+slug: tabstrip-tabs-alignment
+tags: telerik,blazor,tab,strip,tabstrip,alignment
+published: True
+position: 11
+---
+
+# TabStrip Tabs Alignment
+
+By design, the tabs are displayed on the left side of the TabStrip header.
+
+You can customize specify their alignment through the `TabAlignment` parameter. It takes a member of the `Telerik.Blazor.TabStripTabAlignment` enumeration:
```

Comment:
```suggestion
You can customize their alignment through the `TabAlignment` parameter. It takes a member of the `Telerik.Blazor.TabStripTabAlignment` enumeration:
```

Reply:
It might be worth mentioning if and how to control the alignment of the text within the tabs and whether this parameter affects in it any way. 

Also, how does RTL, if enabled, affect the alignment of the tabstrip tabs? Note if it does or doesn't affect alignment.

Reply:
> Also, how does RTL, if enabled, affect the alignment of the tabstrip tabs? Note if it does or doesn't affect alignment.

In general, if the RTL is enabled the change in the behavior of the component feature is implied. So, I don't think we need to document how to feature behavior is changed. In your opinion, if it is recommended/required, we could mention in a note something like - "Right-to-Left Support affects the behavior of alignment.".